packages/styles/progressbar.css

```diff
@@ -18,3 +20,16 @@
   min-width: var(--space-small);
   transition: width linear var(--progress-bar-animation-timing);
 }
+
+.ProgressBar--thin .ProgressBar--fill {
+  height: var(--space-smallest);
```

Comment:
Same issue as above comment. This should be a fixed size and not inheriting from `--space-smallest`.